app/models/wiki.rb

```diff
@@ -73,14 +74,18 @@ def api_url
   #############
 
   def ensure_valid_project
-    # Multilingual projects must have language == nil.
+    # Most multilingual projects must have language == nil,
+    # but wikisource is the exception, it can have nil, or a language.
     # Standard projects must have a language.
-    if MULTILINGUAL_PROJECTS.include?(project)
+    #
+    # TODO: Validate the language/project combination by pinging it's API.
+    if project == "wikidata"
       self.language = nil
+    elsif project == "wikisource"
```

Comment:
To make the `www` convention consistent, we should probably switch on *that* to set language to nil, and save all multilingual project wikis in the database with `language: nil`.